discover/index.html

```diff
@@ -124,5 +124,19 @@ <h3 class="npub w3-large">follow the journey on our nostr:</h3>
   }
 }
 </script>
+<!--- nostri.chat widget by pablof7z--->
+<script
+  src="https://nostri.chat/public/bundle.js"
+  <!-- YOUR PUBKEY IN HEX FORMAT -->
+  data-website-owner-pubkey="YOUR_PUBKEY"
```

Comment:
for `chat-type="GLOBAL"` it would make sense to remove this line

Reply:
Thank you so much for reviewing the nostri.chat code. Would having the PUBKEY there, prevent anyone from writing, except for that npub?